ompi/mca/coll/han/coll_han_alltoallv.c

```diff
@@ -250,16 +338,20 @@ static int alltoallv_sendrecv_w(
         buf_items[jbuf] = opal_free_list_get(&mca_coll_han_component.pack_buffers);
         if (buf_items[jbuf] == NULL) {
             nbufs = jbuf - 1;
-            printf("Uh-oh, not enough buffers: %d\n",nbufs);
+            opal_output_verbose(20, mca_coll_han_component.han_output,
+                "Uh-oh, not enough buffers: %d\n",nbufs);
```

Comment:
ok, but maybe we can make this output a more user-friendly ?

Reply:
Parameterized, and provided help to the user on which knobs to turn.